lib/nostrum/api.ex

```diff
@@ -1156,74 +1169,113 @@ defmodule Nostrum.Api do
     request(:delete, Constants.guild(guild_id))
   end
 
-  @doc """
-  Gets a list of channels.
+  @doc ~S"""
+  Gets a list of guild channels.
+
+  If successful, returns `{:ok, channels}`. Otherwise, returns a `t:Nostrum.Api.error/0`.
+
+  ## Examples
 
-  Guild to get channels for is specified by `guild_id`.
+  ```Elixir
+  Nostrum.Api.get_guild_channels(81384788765712384)
+  {:ok, [%Nostrum.Struct.Channel{} | _]}
```

Comment:
Kind of a meh example IMO, but I don't really have any suggestions to make it better. 🙃

Reply:
If we ever do go around mocking these API requests, these will at least cover the decoding aspect. Maybe this specific example should of had `%Nostrum.Struct.Channel{guild_id: 81384788765712384}` instead of plain `%Nostrum.Struct.Channel{}`.

Reply:
I'm still on the fence of how useful API mocks would be since there's not a whole lot of logic behind them. I guess we can leave this alone for now.

Reply:
API mocks would be useful for testing if payloads are decoded properly. Although, the question of whether the effort is worth the prize is certainly debatable.